Makefile

```diff
@@ -7,7 +7,8 @@ goarch=amd64
 ldXflags=github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=info
 ldXflagsD=github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=debug
 
-MOCKERY_VERSION=v2.38.0
+MOCKERY_VERSION=v2.42.1
```

Comment:
use same Mockery version as in Atlas SDK

Reply:
is this a requirement or more like a good to have so that we don't deviate away from SDK? Asking because I am wondering if we should have some checks in place to ensure this is in alignment with the Atlas SDK version?

Reply:
it's a good to have, but I've created this ticket: https://jira.mongodb.org/browse/CLOUDP-271278